app/views/_container-statuses.html

```diff
@@ -0,0 +1,74 @@
+<!--
+  Expects the following variables:
+  containerStatuses
+  startDebugTerminal (optional)
+  detailed (optional)
+ -->
+
+<div ng-if="detailed && pod.status.initContainerStatuses.length">
+  <h4 row>
+    <span><i class="fa" ng-class="{'fa-check text-success': initContainersTerminated}"></i></span>
+    <span flex>
+      <span ng-if="initContainersTerminated"> &nbsp;{{pod.status.initContainerStatuses.length}}</span>
+      <ng-pluralize count="pod.status.initContainerStatuses.length"
+                    when="{'1': 'Init Container','other': 'Init Containers'}">
```

Comment:
It might be good to use the container name when there's one.

> Init container my-init-container completed successfully

or if more than one like you have

> 3 init containers completed successfully

We usually don't capitalize types in sentences.